gmapping/test/basic_localization_stage_replay.launch

```diff
@@ -4,5 +4,5 @@
   <!--test time-limit="30" test-name="basic_localization_stage" pkg="gmapping" 
         type="test_map.py" args="90.0"/-->
   <!-- TODO: Fix the rtest to work with replay, for now it's not working -->
-  <test time-limit="250" test-name="map_data_test_replay" pkg="gmapping" type="gmapping-rtest" args="200.0 0.05 4000 4000 0.005 0.010"/>
+  <test time-limit="300" test-name="map_data_test_replay" pkg="gmapping" type="gmapping-rtest" args="200.0 0.05 4000 4000 0.005 0.010"/>
```

Comment:
The changes to tf -> tf2 should not change the behavior. tf is using tf2 under the hood these days so if anything it should be slightly faster due to fewer function calls due to no indirection. This suggests that there's some timeouts or delays coming in somewhere.